dev-python/spyder-autopep8/spyder-autopep8-9999.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1 git-r3
+
+DESCRIPTION="Plugin to run the autopep8 python linter from within the spyder editor"
+HOMEPAGE="https://github.com/spyder-ide/spyder-autopep8"
+EGIT_REPO_URI="https://github.com/spyder-ide/spyder-autopep8.git"
```

Comment:
We won't add any "only" live packages to tree, so if this is needed you have to find an archived snapshot tarball and package that as `-0_pre<DateOfLatestCommit>`, ie `0_pre20200125`